src/SmartComponents/BaselinesTable/BaselinesTable.js

```diff
@@ -13,7 +13,7 @@ class BaselinesTable extends Component {
         this.onSelect = this.onSelect.bind(this);
     }
 
-    async componentDidMount() {
+    componentDidMount() {
```

Comment:
I'm not famliar with what `async` did. What does its removal do?

Reply:
When I was talking with Karel a couple of weeks ago, he was saying that the async (asynchronous) wasn't necessary for this. I believe it's because the actual fetch is asynchronous.